sql/load_demo_data.sql

```diff
@@ -1,8 +1,8 @@
--- MariaDB dump 10.19  Distrib 10.6.13-MariaDB, for Win64 (AMD64)
+-- MariaDB dump 10.19  Distrib 10.6.16-MariaDB, for linux-systemd (x86_64)
 --
--- Host: localhost    Database: oh
+-- Host: localhost    Database: ohdemo
```

Comment:
Is this change really intended?

Reply:
actually since now we have only data (since OP-1111) we could remove this comment because pointless
nevertheless, I think it would be added automatically by mysqldump
it's just a comment anyway

Reply:
Yes, let's keep it. It's just a comment but it is better that we enforce the idea of separating real data from demo data.